plugins/out_stackdriver/stackdriver_conf.c

```diff
@@ -286,6 +289,32 @@ struct flb_stackdriver *flb_stackdriver_conf_create(struct flb_output_instance *
         ctx->severity_key = flb_sds_create(tmp);
     }
 
+    tmp = flb_output_get_property("k8s_cluster_name", ins);
+    if (tmp) {
+        ctx->cluster_name = flb_sds_create(tmp);
+    }
+
+    tmp = flb_output_get_property("k8s_cluster_location", ins);
+    if (tmp) {
+        ctx->cluster_location = flb_sds_create(tmp);
+    }
+
+    if (flb_sds_cmp(ctx->resource, "k8s_container", 
+                    flb_sds_len(ctx->resource)) == 0 || 
+        flb_sds_cmp(ctx->resource, "k8s_node", 
+                    flb_sds_len(ctx->resource)) == 0 ||
+        flb_sds_cmp(ctx->resource, "k8s_pod", 
+                    flb_sds_len(ctx->resource)) == 0) {
+        
+        ctx->k8s_resource_type = true;
+        if (!ctx->cluster_name || !ctx->cluster_location) {
+            flb_plg_error(ctx->ins, "Missing k8s_cluster_name "
```

Comment:
missing

nit: on configuration -> in configuration?

Reply:
updated